builder/builder-next/controller.go

```diff
@@ -86,7 +85,7 @@ func newSnapshotterController(ctx context.Context, rt http.RoundTripper, opt Opt
 		return nil, err
 	}
 
-	historyDB, historyConf, err := openHistoryDB(opt.Root, opt.BuilderConfig.History)
+	historyDB, historyConf, err := openHistoryDB(opt.Root, "history_ctd.db", opt.BuilderConfig.History)
```

Comment:
Slight nit; I think we more commonly use `c8d` as short form for containerd (isnteead of `ctd`)


Does the `cache.db` below also depend on which backend is used? I'm wondering if the alternative would be to pass a different `opt.Root` when initialising? Looks like we hard-code `root` to be `/buildkit` (which probably is fine), but perhaps it could use a subdirectory based on what driver is used? https://github.com/moby/moby/blob/83a1693c1609b47fde349756510cbfbf43e34eed/cmd/dockerd/daemon.go#L430

Perhaps even `buildkit/<snapshotterName>` (of this information is tied to the snapshotter used)

Reply:
> Slight nit; I think we more commonly use `c8d` as short form for containerd (isnteead of `ctd`)

Sure! Better to be consistent.

> Does the `cache.db` below also depend on which backend is used? I'm wondering if the alternative would be to pass a different `opt.Root` when initialising? Looks like we hard-code `root` to be `/buildkit` (which probably is fine), but perhaps it could use a subdirectory based on what driver is used?

For cache blobs this is not needed afaik (cc @tonistiigi)